### PR TITLE
[ODS-5502] - Add Postgres Initdev workflows

### PR DESCRIPTION
**Pull request by bptillman**
This establishes two PostgreSQL builds to mirror the previously created initdev builds for SQL Server, one for running DotNet/Integration tests and one for performing Smoke tests. Neither of these Postgres builds package or publish, which mirrors TeamCity where we used the packages created in the SQL Server builds for publishing and do not create any as part of the PostgreSQL build.

Some cleanup was also done on the SQL Server initdev builds to fix an issue with not passing the UsePlugin parameter, and remove anything not needed in each of the builds anymore.

### CODE REVIEW

File: .github/workflows/InitDev (Postgres), Tests using Sandbox, Year-Specific and Shared Instance.yml [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+      uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2
+      with:
+          repository: Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation
+          path: Ed-Fi-ODS-Implementation/
+    - name: Is pull request branch exists in Ed-Fi-ODS-Implementation
+      working-directory: ./Ed-Fi-ODS-Implementation/
+      shell: pwsh
+      run: |
+            $patternName = 'refs/heads/' +  '${{ env.CURRENT_BRANCH }}'
+            Write-Host "Pattern Name is " $patternName -fore GREEN
```
Comment (semalaiappan) [inline .github/workflows/InitDev (Postgres), Tests using Sandbox, Year-Specific and Shared Instance.yml:54]: Can you clean up Write host statement every where in Is pull request branch exists in Ed-Fi-ODS-Implementation step

File: .github/workflows/InitDev (Postgres), Tests using Sandbox, Year-Specific and Shared Instance.yml [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+      uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2
+      with:
+          repository: Ed-Fi-Alliance-OSS/Ed-Fi-ODS
+          path: Ed-Fi-ODS/
+    - name: Is pull request branch exists in Ed-Fi-ODS
+      working-directory: ./Ed-Fi-ODS/
+      shell: pwsh
+      run: |
+           $patternName = 'refs/heads/' +  '${{ env.CURRENT_BRANCH }}'
+           Write-Host "Pattern Name is " $patternName -fore GREEN
```
Comment (semalaiappan) [inline .github/workflows/InitDev (Postgres), Tests using Sandbox, Year-Specific and Shared Instance.yml:71]: Can you clean up Write host statement every where in Is pull request branch exists in Ed-Fi-ODS step

File: .github/workflows/InitDev (Postgres),Unit tests,Integration tests.yml [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+      uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2
+      with:
+          repository: Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation
+          path: Ed-Fi-ODS-Implementation/
+    - name: Is pull request branch exists in Ed-Fi-ODS-Implementation
+      working-directory: ./Ed-Fi-ODS-Implementation/
+      shell: pwsh
+      run: |
+            $patternName = 'refs/heads/' +  '${{ env.CURRENT_BRANCH }}'
+            Write-Host "Pattern Name is " $patternName -fore GREEN
```
Comment (semalaiappan) [inline .github/workflows/InitDev (Postgres),Unit tests,Integration tests.yml:52]: Can you remove  all Write-Host statement in in  Step name: Is pull request branch exists in Ed-Fi-ODS-Implementation

File: .github/workflows/InitDev (Postgres),Unit tests,Integration tests.yml [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+      uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2
+      with:
+          repository: Ed-Fi-Alliance-OSS/Ed-Fi-ODS
+          path: Ed-Fi-ODS/
+    - name: Is pull request branch exists in Ed-Fi-ODS
+      working-directory: ./Ed-Fi-ODS/
+      shell: pwsh
+      run: |
+           $patternName = 'refs/heads/' +  '${{ env.CURRENT_BRANCH }}'
+           Write-Host "Pattern Name is " $patternName -fore GREEN
```
Comment (semalaiappan) [inline .github/workflows/InitDev (Postgres),Unit tests,Integration tests.yml:69]: Can you remove  all Write-Host statement in in  Step name: Is pull request branch exists in Ed-Fi-ODS

File: .github/workflows/InitDev (Postgres),Unit tests,Integration tests.yml [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+                git fetch origin ${{ env.CURRENT_BRANCH }}
+                git checkout ${{ env.CURRENT_BRANCH }}
+              }
+           }
+    - name: Start PostgreSQL
+      run: |
+        $pgService = Get-Service -Name postgresql*
+        Set-Service -InputObject $pgService -Status running -StartupType automatic
+        Start-Process -FilePath "$env:PGBIN\pg_isready" -Wait -PassThru
+    - name: ODS/API InitDev, SdkGen, Integration Test, Unit Test, Package
```
Comment (semalaiappan) [inline .github/workflows/InitDev (Postgres),Unit tests,Integration tests.yml:89]: Can you remove package at the end of name: ODS/API InitDev, SdkGen, Integration Test, Unit Test, Package

**Review by semalaiappan — CHANGES_REQUESTED** [review]
Please fix all review comments

File: .github/workflows/InitDev (Postgres), Tests using Sandbox, Year-Specific and Shared Instance.yml (hunk 1 above)
Comment (bptillman) [reply .github/workflows/InitDev (Postgres), Tests using Sandbox, Year-Specific and Shared Instance.yml]: This has been removed.

File: .github/workflows/InitDev (Postgres), Tests using Sandbox, Year-Specific and Shared Instance.yml (hunk 2 above)
Comment (bptillman) [reply .github/workflows/InitDev (Postgres), Tests using Sandbox, Year-Specific and Shared Instance.yml]: This has been removed

File: .github/workflows/InitDev (Postgres),Unit tests,Integration tests.yml (hunk 3 above)
Comment (bptillman) [reply .github/workflows/InitDev (Postgres),Unit tests,Integration tests.yml]: This has been removed.

File: .github/workflows/InitDev (Postgres),Unit tests,Integration tests.yml (hunk 4 above)
Comment (bptillman) [reply .github/workflows/InitDev (Postgres),Unit tests,Integration tests.yml]: This has been removed

File: .github/workflows/InitDev (Postgres),Unit tests,Integration tests.yml (hunk 5 above)
Comment (bptillman) [reply .github/workflows/InitDev (Postgres),Unit tests,Integration tests.yml]: This has been removed.

**Review by semalaiappan — APPROVED** [review]

**Pull request merged**